argus/action_manager/windows.py

```diff
@@ -283,6 +286,63 @@ def is_dir(self, path):
         """
         return self._exists(path, self.PATH_CONTAINER)
 
+    def _new_item(self, path, item_type):
+        """Create a directory or a file.
+
+        :param path:
+            Instance path of the new item.
+        :param item_type:
+            It can be `Directory` or `File`
+        """
+        cmd = "New-Item -Path '{}' -Type {} -Force".format(path, item_type)
+        self._client.run_command_with_retry(cmd=cmd,
+                                            command_type=util.POWERSHELL)
+
+    def mkdir(self, path):
+        """Create a directory in the instance if the path is valid.
+
+        :param path:
+            Remote path where the new directory should be created.
+        """
+        if self.exists(path):
+            raise exceptions.ArgusCLIError("Directory already exists!")
```

Comment:
You should change the exception message because `self.exists(path)` returns `True` too if `path` points to a file.